components/inputs/input-text.js

```diff
@@ -261,7 +267,7 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 
 	get validity() {
 		const elem = this.shadowRoot.querySelector('.d2l-input');
-		if (!elem.validity.valid) {
+		if (elem && !elem.validity.valid) {
```

Comment:
I am guessing this was causing a JS error in some case?

Reply:
It was, but your comment made me think a bit more about it. The reason it was causing an error was that it was running earlier than before (when the property initializes and before things have rendered vs. after `updated()`).

I've adjusted it so that things should run at the same times as they did before.